crates/cli/src/config.rs

```diff
@@ -67,7 +67,7 @@ pub fn find_rules(
 }
 
 pub fn register_custom_language(config_path: Option<PathBuf>) -> Result<()> {
-  let Ok(mut path) = find_config_path_with_default(config_path, None) else {
+  let Ok(mut path) = find_config_path_with_default(None, config_path.as_deref()) else {
```

Comment:
Hi, thanks for the fix! However, I think the problem is not in the config.rs but instead in the new.rs.

In `run_create_new` function, `register_custom_language` accepts the `base_dir` option instead of the config file.